### PR TITLE
Match new frontpage design

### PR DESCRIPTION
**Pull request by kylegach**
Design: https://www.figma.com/file/ur4kydUbRqdDyfoZWzdiIw/Storybook-app?node-id=1293%3A29554

- Add `@storybook/components-marketing` and `framer-motion` packages
- New Header & Footer
- Refactor AppLayout and how its used with pages
    - Remove `gatsby-plugin-layout`
    - Fetch DX data
- Remove redundant Breadcrumbs
- Various content/typography tweaks
- Move framework and language selectors 
    - From sidebar into subnav

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -2,10 +2,22 @@ const webpack = require('webpack');
...
+  addons: [
+    '@storybook/addon-essentials',
+    '@storybook/addon-a11y',
+    '@storybook/addon-interactions',
+    '@storybook/addon-links',
+  ],
   core: {
     builder: 'webpack5',
   },
+  env: (config) => ({
```
Comment (winkerVSbecks) [inline .storybook/main.js:14]: TIL

**Comment by winkerVSbecks** [comment]
@kylegach not sure why the test command is failing. Maybe use the `--exit-zero-on-changes`  flag

**Comment by winkerVSbecks** [comment]
logos are broken for framework selector. 

Also both the framework and language selector dropdowns are broken. They link to 404. I believe the URL is not constructed correctly. 

CleanShot 2022-09-09 at 15 45 53@2x

File: src/components/screens/ChapterScreen/FrameworkSelector.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  if (framework === 'rax') return '/frameworks/logo-rax.png';
+  return `/frameworks/logo-${framework}.svg`;
+};
+
+const FrameworkLogo = styled.img`
+  width: 12;
+  height: 12;
+`;
+
+function FrameworkSelector({
```
Comment (winkerVSbecks) [inline src/components/screens/ChapterScreen/FrameworkSelector.js:57]: You'll want to use `import { withPrefix } from 'gatsby';`

File: src/components/screens/ChapterScreen/LanguageSelector.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import getChapterInOtherLanguage from '../../../lib/getChapterInOtherLanguage';
+import { GatsbyLinkWrapper } from '../../basics/GatsbyLink';
+
+const getTranslationLanguages = (translationPages) =>
+  translationPages.edges.reduce(
+    (uniqueLanguages, pageEdge) => uniqueLanguages.add(pageEdge.node.fields.language),
+    new Set()
+  );
+
+function LanguageSelector({
```
Comment (winkerVSbecks) [inline src/components/screens/ChapterScreen/LanguageSelector.js:16]: You'll want to use import { withPrefix } from 'gatsby';

**Comment by kylegach** [comment]
> @kylegach not sure why the test command is failing. Maybe use the `--exit-zero-on-changes` flag

I thought I did... 🤔

https://github.com/chromaui/learnstorybook.com/pull/641/files#diff-5db7bb8117a68fd50e5b5e0cbdca28ab4e360e4dd0aeba88911f9404a719700b

**Review by winkerVSbecks — CHANGES_REQUESTED** [review]
Beautiful work! The framework and language selectors need to be fixed. Good to merge after that.

**Comment by kylegach** [comment]
Thanks for catching those missing `withPrefix`s! I believe it's good to go, now.

**Review by winkerVSbecks — APPROVED** [review]

**Comment by kylegach** [comment]
It's not reporting back to the status check, but Netlify _did_ deploy successfully: https://app.netlify.com/sites/storybook-tutorials/deploys/6322375c0dee89571891965d

**Pull request merged**